assets/src/blocks/Accordion/AccordionEditor.js

```diff
@@ -0,0 +1,314 @@
+          accordion_btn_text: attributes.btn_text_1,
+          accordion_btn_url: attributes.btn_url_1
+        };
+        accordion_rows.push(Object.assign({}, accordion_row));
+      }
+
+      if (attributes.accordion_headline.headline_2) {
+        let accordion_row = {
+          accordion_id: attributes.id_2,
+          accordion_headline: attributes.headline_2,
+          accordion_text: attributes.text_2 || '',
+          accordion_btn_text: attributes.btn_text_2,
+          accordion_btn_url: attributes.btn_url_2
+        };
+        accordion_rows.push(Object.assign({}, accordion_row));
+      }
+
+      if (attributes.accordion_headline.headline_3) {
+        let accordion_row = {
+          accordion_id: attributes.id_3,
+          accordion_headline: attributes.headline_3,
+          accordion_text: attributes.text_3 || '',
+          accordion_btn_text: attributes.btn_text_3,
+          accordion_btn_url: attributes.btn_url_3
+        };
+        accordion_rows.push(Object.assign({}, accordion_row));
+      }
+
+      if (attributes.accordion_headline.headline_4) {
+        let accordion_row = {
+          accordion_id: attributes.id_4,
+          accordion_headline: attributes.headline_4,
+          accordion_text: attributes.text_4 || '',
+          accordion_btn_text: attributes.btn_text_4,
+          accordion_btn_url: attributes.btn_url_4
+        };
+        accordion_rows.push(Object.assign({}, accordion_row));
+
+
+      }
+
+      if (attributes.accordion_headline.headline_5) {
+        let accordion_row = {
+          accordion_id: attributes.id_5,
+          accordion_headline: attributes.headline_5,
+          accordion_text: attributes.text_5 || '',
+          accordion_btn_text: attributes.btn_text_5,
+          accordion_btn_url: attributes.btn_url_5
+        };
+        accordion_rows.push(Object.assign({}, accordion_row));
+
+      }
+
+      return accordion_rows;
+    }
+
+    */
+
+
+  //renders the settings
+  renderEdit() {
+    const { attributes } = this.props;
+
+
+    // let { accordion_rows } = attributes;
+
+    // accordion_rows = this.getAccordionRows(accordion_rows);
+
+    return (
+      <Fragment>
+        <InspectorControls>
+          <PanelBody title={__('Settings', 'planet4-blocks-backend')}>
+            <div>
+              <p> Accordion Block. </p>
+            </div>
+          </PanelBody>
+        </InspectorControls>
+
+        {/*
+        {[accordion_rows].map((item, index) => {
+          return (
+            <div key={index}>
+               { !item.accordion_headline && !item.accordion_text &&
+                <div><b>{__('Please provide a title or a description for this accordion row to be displayed.', 'planet4-blocks-backend')}</b></div>
+              }
+
+            </div>
+          );
+        })}
+      */}
+      </Fragment>
+    );
+  }
+
+
+//renders the editor view
+  renderView() {
+    const { attributes, className } = this.props;
```

Comment:
I don't think that this `className` is used anywhere

Reply:
it isn't any more indeed (was used for the multiple style options), removing it